src/components/List/List.tsx

```diff
@@ -160,6 +163,10 @@ function List<T extends Item>({
 
   const showHeader = header || select === 'checkbox' || select === 'switch' || onFilter || onSort;
 
+  const selectAnywhereProps = selectAnywhere
+    ? { tag: Label, style: { cursor: 'pointer', marginBottom: 0 } }
```

Comment:
Could you elaborate on why the 0 margin bottom is needed?

Reply:
The Label tag adds bottom margin styling, so I make it 0 to be consistent with a non-selectAnywhere row